src/progress-indicator/overlay.scss

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+
+@mixin iui-progress-indicator-overlay($backgroundColor: iui-color-background-1, $pointerEvent: auto) {
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  top: 0;
+  left: 0;
+  z-index: 500;
+  pointer-events: $pointerEvent;
```

Comment:
This will handle mouse events but we need to take care of keyboard events too, possibly using some focus trap

Reply:
Shouldnt it be none?